backend/gameplay/consumers.py

```
@@ -197,7 +191,7 @@ async def append_to_position_list(self, chess_game_model: ChessGame, move_info:
 			}
 		})
 
-		await self.update_game_attribute(chess_game_model, "position_list", updated_position_list, should_save=False)
+		chess_game_model.position_list = updated_position_list
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)